src/components/ColorInputFlyout/ColorPickerFlyout.tsx

```diff
@@ -63,6 +65,10 @@ export const ColorPickerFlyout: FC<ColorPickerFlyoutProps> = ({
                         setOpen(false);
                         onClear && onClear();
                     }}
+                    onDelete={() => {
+                        setOpen(false);
+                        onDelete && onDelete();
+                    }}
```

Comment:
If you set it like this, the trash icon will always be displayed because onDelete will never be undefined. Please only pass in the method if onDelete is actually set